docs/source/cookbooks/override-operator-images.md

```diff
@@ -0,0 +1,27 @@
+# Override operator images
+
+## Full deploy override
+In order to override images during an actual deploy, you have to:
+
+* clone [openstack-k8s-operators/openstack-operator](https://github.com/openstack-k8s-operators/openstack-operator/)
+* edit [custom-bundle.Dockerfile](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/custom-bundle.Dockerfile) in order to point to the wanted images
+* run `go mod edit` in the [project root](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/go.mod) and update references
+* run `go mod edit` in the [apis directory](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/apis/go.mod#L11) and update references
+
+Then, you'll have to pass down the repository path to the ci-framework, rebuild it, and deploy:
+```YAML
+cifmw_operator_build_meta_src: /path/to/openstack-operator
+cifmw_operator_build_meta_build: true
+cifmw_operator_build_push_registry: "default-route-openshift-image-registry.apps-crc.testing"
+cifmw_operator_build_push_org: "openstack-operators"
+```
+
+### Example
+You want to update the keystone operator, and point to a specific version.
```

Comment:
At openstack-operators build time currently expected the repo in question is network downloadable.
So local git repos will not be picked up, local only change is not visible.

This method is limited to things which aviailable over network.
 (git can clone from local dir, but if the custom (not meta operator) container dir is not mapped AFAIK)

Reply:
@viroel care to have a look?

Reply:
I think we need to clarify here:
 - user must build and push its own operator images to a registry first
 - user must provide a github api reference in go.mod (I am not sure if path to local changes will work here, needs to be tested)